### ISSUE TITLE
Uninitialized mapping storage pointers should be considered by the static anlysis component

### ISSUE THREAD

**Issue by chriseth** [ethereum/solidity#4641]

**Comment by ekpyron** [comment]
Related: #4427

**Comment by ekpyron** [comment]
The following is already an error (``Type is required to live outside storage.``):
```
function f() public pure returns(mapping(uint=>uint) storage) {}
```
i.e. mappings cannot be returned.
```
function f() public pure { mapping(uint=>uint) storage m; }
```
is also already an error ``Uninitialized mapping. Mappings cannot be created dynamically, you have to assign them from a state variable.``.

So it seems like nothing has to be done here.

**Comment by chriseth** [comment]
Argh, but this is bad, I would consider this to be a bug to be fixed for 0.5.0. New issue: https://github.com/ethereum/solidity/issues/4670

**Comment by ekpyron** [comment]
Yes, I was wondering, whether this was actually ever intended :).
